Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -8343,3 +8343,68 @@
     family: Chrome
   headers:
     http-x-requested-with: com.go.browser
+-
+  user_agent: Mozilla/1.22 (compatible; MSIE 5.01; PalmOS 3.0) EudoraWeb 2.1
+  client:
+    type: browser
+    name: EudoraWeb
+    version: "2.1"
+    engine:
+    engine_version:
+    family:
+-
+  user_agent: Dalvik/2.1.0 (Linux; U; Android 11; VEON 4K Smart TV Build/RTT2.220103.001)
+  client:
+    type: browser
+    name: Basic Web Browser
+    version:
+    engine: WebKit
+    engine_version:
+    family: Android Browser
+  headers:
+    http-x-requested-with: com.apgsolutionsllc.APGSOLUTIONSLLC0007
+-
+  user_agent: Mozilla/5.0 (Android 11; Mobile; rv:107.0) Gecko/107.0 Firefox/107.0 MixerBox-Browser
+  client:
+    type: browser
+    name: MixerBox AI
+    version: ""
+    engine: Gecko
+    engine_version: "107.0"
+    family:
+-
+  user_agent: Mozilla/5.0 (Android 12; Mobile; rv:109.0) Gecko/119.0 Firefox/119.0 YouBrowser/1.4.12
+  client:
+    type: browser
+    name: YouBrowser
+    version: 1.4.12
+    engine: Gecko
+    engine_version: "119.0"
+    family:
+-
+  user_agent: Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/112.0 Firefox/112.0 Freespoke/2.0.0
+  client:
+    type: browser
```

Comment:
This is a search engine. They do have an app (https://play.google.com/store/apps/details?id=com.freespoke&hl=en&gl=US), but I'm not sure I'd call is a browser. I think this should be "mobile app"

Reply:
https://maxbrowser.co/
this program fits the definition of browser

Reply:
I am confused, how is maxbrowser.co relevant here? Max Browser clearly states "browse internet", while Freespoke looks like a news aggregator with search engine of its own, rather than a full-scale web browser. Max Browser has screenshots indicating that you can type URL there, and Freespoke suggests only "search for anything".

Reply:
Should I do any changes?